libraries/template.rb

```diff
@@ -1,8 +1,8 @@
 module Nginx
   module Cookbook
     module TemplateHelpers
-      def nil_or_empty?(v)
-        v.nil? || (v.respond_to?(:empty?) && v.empty?)
+      def nil_or_empty?(*values)
+        values.any? { |v| v.nil? || (v.respond_to?(:empty?) && v.empty?) }
```

Comment:
Guessing this is a Ruby 3.0 bug fix? 

Also thanks for using sensible variable names 😆

Reply:
👌🏼 Nice